docs/stdout_publisher.md

```diff
@@ -1,50 +1,73 @@
-# Example usage with [telegraf]
+# Using the stdout publisher
 
-You can use this project in combination with [telegraf] or similar tools to get
-your inverter's data into other systems than mqtt.
+Enabling this with `DEYE_FEATURE_STDOUT_PUBLISHER` will dump all collected
+metrics in json on stdout once every cycle (every `DEYE_DATA_READ_INTERVAL`
+seconds).
+This can be useful in combination with other tools to process and push the
+metrics into different systems other than mqtt.
 
-An example usage with [telegraf] could look something like that:
+The json object has the following structure:
+```json
+{
+  "serial": "1234567890",
+  "address": "192.168.0.1",
+  "port": 8899,
+  "data": [
+    {
+      "up": 1,
+    }, {
+      "temp": 31,
+      "name": "Radiator temperature",
+      "unit": "°C",
+      "groups": "micro",
+      "sensor": "SingleRegisterSensor",
+      "source": "radiator",
```

Comment:
Imho it would make more sense to rename this property to `sensor`. WDYT?

Reply:
Renamed `source` -> `sensor`